gammapy/data/event_list.py

```diff
@@ -382,22 +384,37 @@ def select_energy(self, energy_range):
         mask &= energy < energy_range[1]
         return self.select_row_subset(mask)
 
-    def select_time(self, time_interval):
-        """Select events in time interval.
+    def select_time(self, time_intervals):
+        """Select events in time intervals.
 
         Parameters
         ----------
-        time_interval : `astropy.time.Time`
+        time_intervals : `astropy.time.Time` or list of time_interval
```

Comment:
If you have a `list of time_interval`, `time_intervals.value` won't work

Reply:
Here I think we probably have to consider checking what is an acceptable group of time intervals.
- do we accept only non overlapping intervals? 
  - If not what has to be done `and` or `or`? I think we should avoid the situation where filtering in a loop over time intervals and filtering over the list yield completely different results.
- what is the expected format of time intervals in general in gammapy? 
  - we are using lists of `Time` objects with shape (2,) in several places already. 
  - we could consider having a specific container for that to limit duplicate checks. After all, `GTI` and `TimeMapAxis` are already such kind of objects